dashboards-observability/public/components/explorer/home.tsx

```diff
@@ -66,15 +69,15 @@ export const Home = (props: IHomeProps) => {
       objectType: ['savedQuery', 'savedVisualization'],
       sortOrder: 'desc',
       fromIndex: 0,
-      maxItems: 10
+//       maxItems: 10
```

Comment:
By default maxItems=1000 if we don't add any number here. Do you want to keep it at default or change to something else. Also can remove the comment if not using it.

Reply:
I'm not using it so i'll remove it